shelf-layouts/Q_Flow_Rundown_View_Layout.json

```diff
@@ -0,0 +1,34 @@
+{
+	"_id": "X4GYdLrEcFTXWKkhm",
+	"name": "Q flow",
```

Comment:
Why `flow`? Isn't this supposed to be the layout of the rundown when in a Q `box`? Unless we are doing something here that is specifically called a "flow", it would be nice if we didn't have multiple names for the same thing.

Reply:
This is what the users (flow producers) are going to see and choose in the dropdown in the Rundown List - I'm assuming it should be what it used to. We could rename the shelf layout, but this wouldn't do much for anyone. We still have different file names, and the layouts are grouped in the settings page.

Reply:
I can follow that logic.
I think however there still might be an argument for calling it box instead of flow. The reason being that it is the layout used when you are in a Qbox no matter if the Qbox is used by a producer, a commentator or a developer.

Reply:
No, a commentator uses a standalone shelf called _Kommentator_